### ISSUE TITLE
Plan for a real back-end

### ISSUE THREAD

**Issue by alanmoo** [mozilla/network-pulse#229]
Let's use this issue as a running list of things that need to be done in order to revamp the back end of Pulse.

Documentation happening at https://docs.google.com/document/d/1nfPzoKSjN25vNVAP-K8WmYB1M2R9Qof5UQ8x5-JJNXM/edit#

- [x] Gather information about existing data structure
- [x] Describe model
- [x] Spin up Django instance
- [ ] ...?

**Comment by alanmoo** [comment]
cc @Pomax

**Comment by xmatthewx** [comment]
Thanks @alanmoo. 

Adding a bit of context: The Pulse prototype is doing well in our pilot test with staff, and was very useful at capturing a slice of MozFest. Very good signals from both MoFo and MoCo that this is useful. We're ready to plan a v1 we can share more broadly. We'll aim for parity with the prototype, plus a few trivial improvements. 

**Users** – For v1, we'll have staff admins. Soon after, we will also need a means for user submissions. Let's draft a user model and then decide how much to bite off initially.

**API** – For v1, we'll just serve items to our own frontend. For v2, we'll likely want a means to serve items for display on other sites, likely linking back to Pulse (e.g. Science or MozFest site show 10 latest items tagged science or mozfest).

**Data** – We have a few adjustments we'll need to make from current structure (particularly around attribution). Nothing very complicated. A few categories, a bunch of strings. 

What's the best way to proceed? A doc to draft and discuss items?

**Comment by xmatthewx** [comment]
Let's discuss this and other big pictures things over video to kick off this work.

**Comment by Pomax** [comment]
Setting up pulse in the spirit of an RSS consumer (projects added to science, etc. this updates their syndicated list of things, the syndication is registered with Pulse, it then checks every X hours/days for an update) feels like a nice approach

**Comment by xmatthewx** [comment]
We will focus first on input by staff and the first ring of people in our network. Sharing well is an art. 

Last spring, I manually extracted all the science collaborate projects and posted to Pulse as if I were a machine. Migration, without curation and editing. I deleted all those entries when Pulse hit the main stage at MozFest with autoscroll by Alan. The projects might be interesting, but the entries were dry, uninspiring, and killing the good vibes.
